st2common/st2common/util/config_loader.py

```diff
@@ -98,6 +100,20 @@ def _get_values_for_config(self, config_schema_db, config_db):
         config = self._assign_default_values(schema=schema_values, config=config)
         return config
 
+    @staticmethod
+    def _get_object_property_schema(object_schema, init_additional_properties=None):
```

Comment:
Please add docstring which explains what this function + ``init_additional_properties`` argument does.

Reply:
done